### DISCUSSION TITLE
[Mobile] wrong cursor with external keyboard

### DISCUSSION THREAD

**Discussion by aleripe** [singerdmx/flutter-quill#408] [Mobile]
My issue is about [Mobile]

I have tried running `example` directory successfully before creating an issue here. Latest Flutter version in stable channel.

Sony Xperia Tablet with Android 5.1.1 and a bluetooth external keyboard. Running the examplate directory. I type some text then press backspace some times. When I try to insert some other text cursor position is wrong, usually a random number of characters after.

**Comment by singerdmx** [comment]
Please submit a PR to fix it

**Comment by aleripe** [comment]
I'm not able to fix it
